### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`mixin` -> `@apply`
`version++`
`polymer++`

### CODE REVIEW

**Comment by cdata** [comment]
https://github.com/PolymerElements/iron-overlay-behavior/blob/7313de07eba31e7b39c4da36f24ac47219d061d7/iron-overlay-backdrop.html#L18

This should be changed to an inline default value.

File: bower.json [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
-    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.8.0",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#^0.8.0",
-    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.9.0",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.9.0",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#^0.9.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.9.0",
     "web-component-tester": "*",
     "webcomponentsjs": "^0.6.0"
```
Comment (cdata) [inline bower.json]: I think this is supposed to be `webcomponents/webcomponentsjs#^0.6.0`, but I'm not sure that it matters.

Comment (notwaldorf) [inline bower.json]: Did it anyway.

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**